bindings/python/setup.py

```diff
@@ -89,20 +94,31 @@ def write_b2_python_config(target, config):
     paths = sysconfig.get_paths()
     includes = [paths["include"], paths["platinclude"]]
 
-    # Note that on debian, the extension suffix is overwritten, but it's
-    # necessary everywhere else, or else b2 will just build "libtorrent.so".
-    # python.jam appends SHLIB_SUFFIX on its own.
-    target = target.name.split("libtorrent")[1]
-    if os.name == "nt":
-        suffix = sysconfig.get_config_var("EXT_SUFFIX")
-    else:
-        suffix = sysconfig.get_config_var("SHLIB_SUFFIX")
-    if suffix != None and target.endswith(suffix):
-        target = target[:-len(suffix)]
-
-    using = f'using python : {sysconfig.get_python_version()} : {escape(sys.executable)} : {" ".join(escape(path) for path in includes)} : : <libtorrent-python>on : "{target}" ;\n'
-    print(using)
```

Comment:
I like being able to see this part of the configuration in the CI log

Reply:
This is still done here:

https://github.com/AllSeeingEyeTolledEweSew/libtorrent/blob/3e2c28b7bb609e8ae5c2086566c0aabb725794b8/bindings/python/setup.py#L230-L231

It's just only output when debugging. I was finding that I really needed to see all b2's debug output to understand how it was digesting the config, for example whether it was getting overridden by something in `user-config.jam`. Should I make this an unconditional print? Or bump some debug level in CI? Bumping to the max looks like this right now:

```bash
DISTUTILS_DEBUG=1 python setup.py -v
```

Reply:
well, it would be useful to see this config on the CI. But I don't think the verbose `b2` debug output is very useful there.

Reply:
Sounds good. I'll add it as an unconditional log